Demo/Color+Helpers.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIColor {
```

Comment:
I like the hex though! I can get it straight from the designs of the project. I's nice to have the demo applied to the theme.

But adding SweetUIKit via carthage i get:
```
Library not loaded: @rpath/SweetUIKit.framework/SweetUIKit
Referenced from: /var/containers/Bundle/Application/B7568C16-DAF3-4C29-9E48-CC36ADA33253/Demo.app/Demo
Reason: image not found
```